critiquebrainz/ws/review/views.py

```diff
@@ -34,8 +34,8 @@ def get_review_or_404(review_id):
     return review
 
 
-@review_bp.route('/<uuid:review_id>', methods=['GET'])
-@crossdomain()
+@review_bp.route('/<uuid:review_id>', methods=['GET', 'OPTIONS'])
+@crossdomain(headers="Authorization, Content-Type")
```

Comment:
The same url here is mapped to GET and OPTIONS here. Further, down to POST and OPTIONS so on. This can be problematic as OPTIONS has multiple bindings. I'll look into this to see what the best way to handle this.